### PR TITLE
Implement CharacterSpacing property in LabelHandlers

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Implement `CharacterSpacing` property in LabelHandlers

### Platforms Affected ### 

- Core
- iOS
- Android

### PR Checklist ###

- [x] Targets the correct branch 
- [x] Tests are passing (or failures are unrelated)
- [x] Targets a single property for a single control (or intertwined few properties)
- [x] Adds the property to the appropriate interface
- [x] Avoids any changes not essential to the handler property
- [x] Adds the mapping to the PropertyMapper in the handler
- [x] Adds the mapping method to the Android, iOS, and Standard aspects of the handler
- [x] Implements the actual property updates (usually in extension methods in the Platform section of Core)
- [x] Tags ported renderer methods with [PortHandler]
- [x] Adds an example of the property to the sample project (MainPage)
- [x] Adds the property to the stub class
- [x] Implements basic property tests in DeviceTests

### CODE REVIEW

**Review by mattleibow — COMMENTED** [review]
Seems OK, just not sure if I understand the way iOS works.

File: src/Core/src/Platform/iOS/CharacterSpacingExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+					return true;
+
+			return false;
+		}
+
+		internal static void AddKerningAdjustment(NSMutableAttributedString mutableAttributedString, string? text, double characterSpacing)
+		{
+			if (!string.IsNullOrEmpty(text))
+			{
+				if (characterSpacing == 0 && !mutableAttributedString.HasCharacterAdjustment())
```
[the remark this answers is not in the document]
Comment (mattleibow) [reply src/Core/src/Platform/iOS/CharacterSpacingExtensions.cs]: I did not understand the logic like a dummy.

File: src/Core/src/Platform/iOS/CharacterSpacingExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,58 @@
+﻿using Foundation;
+using UIKit;
+
+namespace Microsoft.Maui
+{
+	public static class CharacterSpacingExtensions
+	{
+		public static NSMutableAttributedString? AddCharacterSpacing(this NSAttributedString attributedString, string text, double characterSpacing)
```
[the remark this answers is not in the document]
Comment (mattleibow) [reply src/Core/src/Platform/iOS/CharacterSpacingExtensions.cs]: wut?

File: src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.Android.cs [hunk 3]
```diff
@@ -9,6 +9,32 @@ namespace Microsoft.Maui.DeviceTests
 {
 	public partial class LabelHandlerTests
 	{
+		[Fact(DisplayName = "CharacterSpacing Initializes Correctly")]
+		public async Task CharacterSpacingInitializesCorrectly()
```
[the remark this answers is not in the document]
Comment (mattleibow) [reply src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.Android.cs]: Added some iOS tests

**Review by mattleibow — APPROVED** [review]

**Review by hartez — APPROVED** [review]

**Pull request merged**